### ISSUE TITLE
Regarding SpaceNet dataset split

### ISSUE THREAD

**Issue by letme-hj** [sustainlab-group/SatMAE#8]
Thank you for your work and sharing code!

I want to ask for your help regarding the dataset split.
I am currently trying to work on semantic segmentation with SpaceNet v1 dataset, 
but I couldn't find the original split of the dataset.

The SpaceNet v1 dataset I have contains 6940 train samples with geojson files.
I believe that you used 5000 of them as train set, and the rest 1940 as test set for evaluation (as written on the paper). Is this correct?

If it is, may I ask where I can get the information regarding the original split info that you referred to in https://github.com/sustainlab-group/SatMAE/issues/3 ? Or any other suggestions?

Thank you in advance!

**Comment by sauryeo** [comment]
May I ask if you have already implemented fine-tuning for the segmentation task?
